buildTestImages.ts

```diff
@@ -9,12 +8,13 @@ import { refreshDockerfile } from './makeDockerfile';
 refreshDockerfile();
 
 const options = {
+  buildArgs: { type: 'string' },
   match: { short: 'm', type: 'string' },
   dry: { type: 'boolean' },
-};
+} as const;
 const { values } = parseArgs({ options });
 
-const { match, dry } = values;
+const { buildArgs, match, dry } = values;
```

Comment:
Need a better name. `buildArgs` is easy to confuse with Docker's own [Build arguments](https://docs.docker.com/build/guide/build-args/)